assets/css/src/components/widgets.css

```diff
@@ -23,7 +23,11 @@
 	& a {
 		margin-left: 2px;
 		margin-right: 2px;
-		text-decoration: none;
```

Comment:
@emeaguiar Links in widgets did not have a way to identify as links, added text-decoration and removing on hover/active.

Reply:
Doesn't seem to be removed on hover on my side, I'll make the update